setup.py

```diff
@@ -64,6 +64,9 @@
     "pypdfium2",
     "pytest",
     "pytest-asyncio",
+    "thefuzz",
+    "nltk",
+    "alt-profanity-check",
```

Comment:
You should be able to include `SUMMARY_REQUIREMENTS` and `PROFANITY_REQUIREMENTS` instead of hardcoding these.  Might have to move `PROFANITY_REQUIREMENTS` above `DEV_REQUIREMENTS`

Reply:
ohh didn't see that, let me add those